index.html

```diff
@@ -12,88 +12,196 @@
     />
   </head>
   <body>
-    <div class="container">
-      <section class="Content">
-        <h1 class="about-me">About Myself!</h1>
+    <!-- First Figma-->
+    <section class="card">
+      <img src="Asset/Tonic.png" alt="Tonic Company" class="tonic-image" />
+      <div class="TonicHeader">
+        <p>Tonic</p>
+      </div>
+      <div class="HeaderTitle">
+        <p class="tonic-title">CANOPY</p>
+        <img src="Asset/Counter.png" class="counter" />
+        <p class="TonicEnd">Back End Dev</p>
+        <img src="Asset/Counter.png" class="counter" />
+        <p class="TonicEnd">2015</p>
+      </div>
+      <div class="Content">
+        <p>
+          a daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+      </div>
+      <div class="ButtonGroup">
+        <button>html</button> <button>css</button> <button>javaScript</button>
+      </div>
+      <div class="See-more">
+        <button>See Project</button>
+      </div>
+    </section>
+    <!--Second Figma-->
+    <section class="card">
+      <img src="Asset/MultiPost.png" alt="Tonic Company" class="tonic-image" />
+      <div class="TonicHeader">
+        <p>Multi-Post Stories</p>
+      </div>
+      <div class="HeaderTitle">
+        <p class="tonic-title">CANOPY</p>
+        <img src="Asset/Counter.png" class="counter" />
+        <p class="TonicEnd">Back End Dev</p>
+        <img src="Asset/Counter.png" class="counter" />
+        <p class="TonicEnd">2015</p>
+      </div>
+      <div class="Content">
+        <p>
+          a daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+      </div>
+      <div class="ButtonGroup">
+        <button>html</button> <button>css</button> <button>javaScript</button>
+      </div>
+      <div class="See-more">
+        <button>See Project</button>
+      </div>
```

Comment:
- [ ] Here too I would suggest you to remove `div` tag. If you want to style button tag just use button or class name or id to style it but including one element in a `div` tag serves no purpose it just makes your code messy and difficult to style.

Note: I would suggest to remove extra `div` tags from all the cards. These suggestions apply to all your cards.